core/domain/voiceover_domain_test.py

```diff
@@ -167,7 +167,7 @@ def test_validate_language_accent_code_pattern(self) -> None:
             self.entity_voiceovers_instance.validate()
 
         with self.assertRaisesRegex(utils.ValidationError, expected_error_msg):
-            self.entity_voiceovers_instance.language_accent_code = 'en-us'
+            self.entity_voiceovers_instance.language_accent_code = 'en-'
```

Comment:
Hi, @lkbhitesh07 we do need this change since we require a negative test case to ensure the updated regex is working. Earlier one i.e., 'en-us' will pass the new regex. Does this help?
Thank you